expo/gbe/views/class_changestate_view.py

```diff
@@ -17,5 +17,11 @@ def bid_state_change(self, request):
         # if the class has been rejected/no decision, clear any schedule items.
         if request.POST['accepted'] not in ('2', '3'):
             self.object.scheduler_events.all().delete()
+        else:
+            # We have to keep b_ and e_ data consistent somehow
+            # this seems like a good point, as bid editing and event
+            # editing are quite different
+            self.object.e_title = self.object.b_title
+            self.object.e_description = self.object.b_description
```

Comment:
When the class is accepted - we have to do this or it wont' show up right in scheduling.  Also - this seemed like the transition point between bids (and bid editing) and classes and class editing.